src/versionbits.cpp

```diff
@@ -136,6 +136,27 @@ class VersionBitsConditionChecker : public AbstractThresholdConditionChecker {
 
 }
 
+bool IsVersionBitRejected(const Consensus::Params& params, Consensus::DeploymentPos pos){
+
+    bool isRejected = false;
+
+    std::vector<std::string>& versionBitVotes = mapMultiArgs["-rejectversionbit"];
+
+    int bitTest = params.vDeployments[pos].bit;
+
+     BOOST_FOREACH(std::string rejectedBit, versionBitVotes) {
+       int rBit =  stoi(rejectedBit);
```

Comment:
what happens if i don't have an int string in my config eg. rejectversionbit=A

Do we care?

Reply:
im not to sure i can try, catch it or something? again @aguycalled

Reply:
it would be interpreted as 0. isdigit() can be used as a very simple check, other than that can be unnecessarily resource-consuming
http://www.cplusplus.com/reference/cctype/isdigit/